runtime/lua/vim/treesitter.lua

```diff
@@ -73,16 +73,16 @@ function M._create_parser(bufnr, lang, opts)
   return self
 end
 
---- Gets the parser for this bufnr / ft combination.
+--- Gets the parser for a specific buffer and filetype.
 ---
 --- If needed this will create the parser.
 --- Unconditionally attach the provided callback
```

Comment:
I think we can remove this line as it is pretty disturbing.

Reply:
But it's true, right?